raml-path-selector.html

```diff
@@ -175,10 +175,8 @@
     <style>
     :host {
       display: block;
-
-      background-color: var(--raml-path-selector-background-color, --primary-background-color);
```

Comment:
Second argument for `var()` css function is default value. It is better to do it as `var(--raml-path-selector-background-color, transparent)` or even `var(--raml-path-selector-background-color, var(--primary-background-color, transparent))`. This way it's more scalable.

Reply:
@jarrodek that makes sense, thank you.